packages/fiori/src/ShellBarItem.ts

```diff
@@ -79,8 +82,18 @@ class ShellBarItem extends UI5Element {
 	get stableDomRef() {
 		return this.getAttribute("stable-dom-ref") || `${this._id}-stable-dom-ref`;
 	}
+
+	fireClickEvent(e: MouseEvent) {
+		this.fireEvent<ShellBarClickEventDetail>("click", {
```

Comment:
if you don't return the value returned by fireEvent, we won't get the correct value 
in:
`const prevented = shellbarItem!.fireClickEvent(e);`

I will check if we have the same error in the NotificationAction

Reply:
There, it's the same - no return, but at least noone checks the returned value, while here it immediately causes a bug.